src/main/resources/redmine-attachments/output_schema/getAttachment.json

```diff
@@ -0,0 +1,56 @@
+{
```

Comment:
We need to put licencing headers for schema as well. Is there any technical blocker?